src/components/Explorer/Base/Result/RunLookup/PalmPrintsTable.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react'
+import * as Utils from '../../../../../common/utils'
+import { useEffect, useState } from 'react'
+import { NavLink } from 'react-router-dom'
+import { routes } from '../../../../../common/routes'
+
+interface PalmPrintsTableProps {
+    data: any[] | undefined
+    header: { [key: string]: string }
+}
+
+export const PalmPrintsTable = ({ data, header }: PalmPrintsTableProps) => {
+    const [filteredData, setFilteredData] = useState<any>([])
+    useEffect(() => {
+        if (data && data.length > 0) {
+            setFilteredData(Utils.filterObject(data, Object.keys(header)))

```

Comment:
Instead of having a separate `Utils.filterObject`, you can do something like [this](https://stackoverflow.com/a/56081419):

```js
# filter data by keys in header
Object.fromEntries(
   Object.entries(data).filter(
      ([key, val])=>Object.keys(header).includes(key)
   )
)
```

Reply:
This looks cleaner but I opted the other solution mainly for its readability and reusability for novice JS devs too who might look at this code at some point.